spec/spec_helper.rb

```diff
@@ -107,6 +107,12 @@
 
   config.before { Redis.new.flushdb }
 
+  # Temporarily disable the reference_selection feature. This flag will be
+  # selectively enabled as the feature is being built out. The line below will
+  # be removed prior to switching the feature on in production. See:
+  # https://trello.com/c/hZVGCcxd
+  config.before { FeatureFlag.deactivate(:reference_selection) }
```

Comment:
Here i would manually deactivate this is the specs that fail due to it. Then we know which specs to fix when we pull the flag out.

Reply:
Taking care of this currently as part of app submit, where it's easier to do once a complete flow under the new references design is implemented. At this stage, ~40 specs fail with the feature on.